docs/reference/cat/recovery.asciidoc

```diff
@@ -12,17 +12,22 @@ way for shards to be loaded from disk when a node starts up.
 As an example, here is what the recovery state of a cluster may look like when there
 are no shards in transit from one node to another:
 
-[source,sh]
+[source,text]
```

Comment:
We use `source,js` for these. We might should use `source,console` but we use `source,js`. These are just markers for us. The syntax highlighter infers the language at page load.